dace/dtypes.py

```diff
@@ -247,10 +247,22 @@ def __init__(self, wrapped_type):
                 wrapped_type = getattr(numpy, wrapped_type)
             except AttributeError:
                 raise ValueError("Unknown type: {}".format(wrapped_type))
+
+        config_data_types = Config.get('compiler', 'default_data_types')
         if wrapped_type is int:
-            wrapped_type = numpy.int64
+            if config_data_types == 'Python default':
```

Comment:
I would check `config_data_types.lower() == 'python'` to be sure with weird configuration inputs

Reply:
applies to the rest too